.gitignore

```diff
@@ -21,3 +21,6 @@
 /yarn-error.log
 
 .byebug_history
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
What is this for. I've never heard of `application.yml`. I see you have an `application.rb`.